source/applications/advanced/gamma_correction.py

```diff
@@ -0,0 +1,42 @@
+# Gamma adjustment
+import numpy as np
+import cv2
+from pathlib import Path
+import zivid
+
+
+def adjust_gamma(image, gamma=1.0):
+    # build a lookup table mapping the pixel values [0, 255] to
+    # their adjusted gamma values
+    invGamma = 1.0 / gamma
+    table = np.array(
+        [((i / 255.0) ** invGamma) * 255 for i in np.arange(0, 256)]
+    ).astype("uint8")
+    # apply gamma correction using the lookup table
+    return cv2.LUT(image, table)
+
+
+def _load_zivid_image():
+    app = zivid.Application()
```

Comment:
We usually initialize zivid application in main(). Consider moving this line to beginning of main()

Reply:
Ok, I will move it!